garnbarn_api/serializer.py

```diff
@@ -109,6 +118,7 @@ class CreateAssignmentApiSerializer(serializers.ModelSerializer):
     reminderTime = ReminderTimeField(source='reminder_time', default=None,
                                      child=serializers.IntegerField()
                                      )
+    tag = CreateTagApiSerializer(default=None)
```

Comment:
Users shouldn't specify the Tag object from the API. But instead, they have to specify Tag ID.

Currently, I can still specify the tag param in the create assignment API request. But the server return 500

Example: 
```
{
    "name": "Test1234",
    "description": "Here!",
    "tag": {
        "test": "A",
        "name": "1234"
    }
}
```

Reply:
resolved with https://github.com/GarnBarn/garnbarn-backend/pull/30/commits/8e996c3dec4f89ae8582ce077278c6b538b07b70

Reply:
Fixed.